ufm-test-services/.env.example

```diff
@@ -5,14 +5,24 @@ CI=false
 GRAFANA_ADMIN_PWD=op
 
 # Used by Test Services to query metrics. http://prometheus will use Docker's built-in DNS
-PROMETHEUS_SERVER_URL="http://prometheus:9090"
+METRICS_READ_URL="http://prometheus:9090/api/v1/query"
+# The needed credentials to access METRICS_READ_URL. Will be sent as: Authorization: Bearer username:password
+METRICS_READ_USERNAME=""
+METRICS_READ_PASSWORD=""
 
 # Used by Test Services to push metrics. http://pushgateway will use Docker's built-in DNS
-PROMETHEUS_PUSHGATEWAY_URL="http://pushgateway:9091"
-
-# If true, Metamask Test Service will install Xvfb inside it's container and used that for Playwright tests.
-# If false, Metamask you will need to specify METAMASK_DISPLAY and METAMASK_DISPLAY_VOLUME so Playwright can connect to a display
-METAMASK_RUN_HEADLESS=true
+METRICS_WRITE_URL="http://pushgateway:9091"
+# Dictates how the request body is structured when pushing metrics. Should be either "grafana" or "prometheus-pushgateway"
+METRICS_WRITE_TOOL="prometheus-pushgateway"
+# This is the source the pushed metric will be labeled as originting from. May not need this value
+METRIC_WRITE_SOURCE=""
+# The needed credentials to access METRICS_READ_URL. Will be sent as: Authorization: Bearer username:password
+METRICS_WRITE_USERNAME=""
```

Comment:
q: do the write credentials also work as read credentials? Wondering if we could consolidate the two

Reply:
No they won't. There might be instances (for other OP chains) where they do, but for our use case they're different